<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="formLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>OS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>TcOS</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_2">
              <property name="toolTip">
               <string>取操作系统的语言，仅支持 Windows</string>
              </property>
              <property name="text">
               <string>取操作系统的语言</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="toolTip">
               <string>取操作系统名称，支持Windows和Linux</string>
              </property>
              <property name="text">
               <string>取操作系统名称</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_5">
           <attribute name="title">
            <string>TcChinese</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_10">
              <property name="toolTip">
               <string>中文名字翻译为英文习惯</string>
              </property>
              <property name="text">
               <string>中文名字翻译为英文习惯</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_9">
              <property name="toolTip">
               <string>混合字符时，转换中不过滤英数符号等</string>
              </property>
              <property name="text">
               <string>混合字符时，转换中不过滤英数符号等</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_8">
              <property name="toolTip">
               <string>取得多个汉字的全部拼音</string>
              </property>
              <property name="text">
               <string>取得多个汉字的全部拼音</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_7">
              <property name="toolTip">
               <string>取得多个汉字的首字母</string>
              </property>
              <property name="text">
               <string>取得多个汉字的首字母</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>TcUdp</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_6">
              <property name="toolTip">
               <string>向指定的端口发送广播字符串。不需要启动为服务器</string>
              </property>
              <property name="text">
               <string>Udp广播3次</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_4">
              <property name="toolTip">
               <string>关闭Udp服务器</string>
              </property>
              <property name="text">
               <string>关闭Udp服务器</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="toolTip">
               <string>向本地服务器发送字符集</string>
              </property>
              <property name="text">
               <string>向本地服务器发送3次字符集</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_3">
              <property name="toolTip">
               <string>启动为可接收Udp信息的服务器</string>
              </property>
              <property name="text">
               <string>启动Udp服务器</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="5" column="0" colspan="3">
             <widget class="QTextEdit" name="udpMessages"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Component</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TcDateEdit</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="TcDateEdit" name="tcDateEdit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>System</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget_4">
          <widget class="QWidget" name="tab_9">
           <attribute name="title">
            <string>Qt-Installer-Framework</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="4" column="0">
             <widget class="QPushButton" name="pushButton_16">
              <property name="toolTip">
               <string>文件系统类型</string>
              </property>
              <property name="text">
               <string>文件系统类型</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_12">
              <property name="toolTip">
               <string>判断当前运行的程序有没有管理员权限</string>
              </property>
              <property name="text">
               <string>判断当前运行的程序有没有管理员权限</string>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QPushButton" name="pushButton_19">
              <property name="toolTip">
               <string>可用空间，单位字节</string>
              </property>
              <property name="text">
               <string>可用空间，单位字节</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_18">
              <property name="toolTip">
               <string>分区大小,单位字节</string>
              </property>
              <property name="text">
               <string>分区大小,单位字节</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_14">
              <property name="toolTip">
               <string>获取路径（C:）所在的分区信息</string>
              </property>
              <property name="text">
               <string>获取路径（C:\）所在的分区信息</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="pushButton_17">
              <property name="toolTip">
               <string>分区描述符</string>
              </property>
              <property name="text">
               <string>分区描述符</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_13">
              <property name="toolTip">
               <string>以管理员身份启动新的程序</string>
              </property>
              <property name="text">
               <string>以管理员身份启动新的程序</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushButton_15">
              <property name="toolTip">
               <string>挂载的path，win下为盘符，即C:/</string>
              </property>
              <property name="text">
               <string>挂载的path，win下为盘符，即C:/</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_22">
              <property name="toolTip">
               <string>正在运行的进程列表</string>
              </property>
              <property name="text">
               <string>正在运行的进程列表</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pushButton_23">
              <property name="toolTip">
               <string>杀死记事本进程</string>
              </property>
              <property name="text">
               <string>杀死记事本进程</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_21">
              <property name="toolTip">
               <string>当前系统的分区信息</string>
              </property>
              <property name="text">
               <string>当前系统的分区信息</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1">
             <widget class="QPushButton" name="pushButton_24">
              <property name="toolTip">
               <string>检查路径是不是在本地磁盘上</string>
              </property>
              <property name="text">
               <string>检查路径是不是在本地磁盘上</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_20">
              <property name="toolTip">
               <string>当前系统的内存大小</string>
              </property>
              <property name="text">
               <string>当前系统的内存大小</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_25">
              <property name="text">
               <string>在桌面创建快捷方式</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_10">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>其它示例</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>浮点数转金额</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_26">
          <property name="text">
           <string>截屏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string>截屏效果显示区（拉伸效果）</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" name="ButtonWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>演示翻页按钮=&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>547</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>AES 加密</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>明文</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>结果</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="edDesPwd"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_27">
          <property name="text">
           <string>加密 / 解密</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>601</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>5</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TcDateEdit</class>
   <extends>QComboBox</extends>
   <header>tcDateEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
